File: app/src/main/java/com/casebeaumonde/activities/eventDetail/response/AddItemToAnotherCloset.java
package com.casebeaumonde.activities.eventDetail.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AddItemToAnotherCloset {

@SerializedName("code")
@Expose
private String code;
@SerializedName("messsage")
@Expose
private String messsage;

public String getCode() {
return code;
}

public void setCode(String code) {
this.code = code;
}

public String getMesssage() {
return messsage;
}

public void setMesssage(String messsage) {
this.messsage = messsage;
}

}

File: app/src/main/java/com/casebeaumonde/activities/eventDetail/response/AddItemToAnotherCloset.kt
package com.casebeaumonde.activities.eventDetail.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AddItemToAnotherCloset {

    @SerializedName("code")
    @Expose
    private var code: String? = null

    @SerializedName("messsage")
    @Expose
    private var messsage: String? = null

    fun getCode(): String? {
        return code
    }

    fun setCode(code: String?) {
        this.code = code
    }

    fun getMesssage(): String? {
        return messsage
    }

    fun setMesssage(messsage: String?) {
        this.messsage = messsage
    }

}